### ISSUE TITLE
add Install instructions

### ISSUE THREAD

**Issue by farrukhaskari1** [casparwylie/Astral#1]
add install instructions and required frameworks and files and way to add them

**Comment by ghost** [comment]
Dude, I want to follow the project. How can I do it?

**Comment by casparwylie** [comment]
Right now, I am spending hours getting it ready for launch in a week. Once it is settled on the app store, I am going to start providing information for those who want to contribute.

**Comment by farrukhaskari1** [comment]
anu update ?

**Comment by casparwylie** [comment]
Yes. App is in review. Launch is very close. Maybe one or two more weeks. Will be sure to tell you, then I'll add details.

**Comment by ghost** [comment]
any update?
